official-git/fix-ignores-test.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/t/t0008-ignores.sh b/t/t0008-ignores.sh
+index 89544dd83..217068733 100755
+--- a/t/t0008-ignores.sh
++++ b/t/t0008-ignores.sh
+@@ -187,7 +187,7 @@ test_expect_success 'setup' '
+ 		git init &&
+ 		echo a >a &&
+ 		git add a &&
+-		git commit -m"commit in submodule"
```

Comment:
There's no space between the -m and the commit message, which causes go's flags package to fail on it and also the git manual specifies a space between -m and the message. It seems that the official git command-line just supports this unspecified syntax and the test here happens to use it.